src/Core/src/Platform/Android/LocalizedDigitsKeyListener.cs

```diff
@@ -0,0 +1,224 @@
+﻿using System.Collections.Generic;
+using Android.Text;
+using Android.Text.Method;
+using Java.Lang;
+using Java.Text;
+
+namespace Microsoft.Maui.Platform.Android
+{
+	internal class LocalizedDigitsKeyListener : NumberKeyListener
```

Comment:
If this is a copy from Compatibility, then maybe we should remove that one and use this one there as well?